Misc/NEWS.d/next/Library/2021-08-16-17-52-26.bpo-44850.r8jx5u.rst

```diff
@@ -0,0 +1,3 @@
+The performance of ``operator.methodcaller`` has been improved by using
+``_PyObject_GetMethod`` (avoiding the creation of a bound method in many
```

Comment:
The NEWS  seems outdated. Benchmark results are besed on `_PyObject_GetMethod`, which is not in the diff.
And it might be better to rewrite this PR's introduction (which now looks historically complicated).

Reply:
thanks, edited.